ansible_ai_connect/ai/api/telemetry/schema1.py

```diff
@@ -180,29 +180,20 @@ class ChatBotBaseEvent:
         converter=str,
         default=settings.CHATBOT_DEFAULT_PROVIDER,
     )
-    modelName: str = field(
-        validator=validators.instance_of(str), converter=str, default=settings.CHATBOT_DEFAULT_MODEL
-    )
-    rh_user_org_id: int = field(validator=validators.instance_of(int), converter=int, default=-1)
-    timestamp: str = field(
-        default=Factory(lambda self: timezone.now().isoformat(), takes_self=True)
-    )
 
     def __attrs_post_init__(self):
```

Comment:
This function isn't really used after the refactor as the properties are not set at the time of instantiation.

The `attrs` library calls this after the `__init__` function.

Reply:
I am not sure, I think @goneri implemented this, let's see if he can give a pointer here... :)

Reply:
- `timestamp` is set line 90
- `rh_user_org_id` is set by `set_user()`
- `modelName` is a bit special because it should be set from `AACSAPIView` ([See](https://github.com/ansible/ansible-ai-connect-service/blob/main/ansible_ai_connect/ai/api/views.py#L226)). So it's better to set if from the view as an attribute.